### ISSUE TITLE
The Tweets with only images are not recognized when asked to LLM

### ISSUE THREAD

**Issue by tomipardinas** [supermemoryai/supermemory#156]
When asked about a specific text from the database, it only recognizes text, but the text from the image isn't recognized:
Here the question:
<img width="792" alt="image" src="https://github.com/user-attachments/assets/30a956d8-1fd7-44f5-ae9e-2fc1bc3e2095">

Here the tweet mentioning the image with the text
<img width="589" alt="image" src="https://github.com/user-attachments/assets/708feaad-8e56-46c8-8f17-10a739946ee3">

**Comment by Dhravya** [comment]
There's ways to fix this right now, but I think that at scale it doesn't make sense to use a vision model or even OCR for that matter on every single image.
We currently are reaching into 60,000 tweets imported into supermemory. a _lot_ of them also have images. would get expeensive

**Comment by CodeTorso** [comment]
so you have any other method method ?

**Comment by krakenftw** [comment]
@Dhravya for the time being, if we really want the feature, we can use some OCR API? which are free/extremely cheap?

**Comment by krakenftw** [comment]
https://ocr.space/ocrapi here is one

**Comment by Welding-Torch** [comment]
This should be tackled eventually. If we're importing tweets into Supermemory, providing a first-class experience to the user is important.

**Comment by tomipardinas** [comment]
Yes. If the objective is truly to create a "super" memory, it needs to be able to extract useful information from all types of unstructured data in your tweets, including images containing text. This is especially important because tweets with images are more likely to be retweeted, which increases their chances of being valuable and, consequently, the probability of being bookmarked.

**Comment by Dhravya** [comment]
Hmmm. this is gonna be a long discussion. We did create endpoints and stuff to make this work long ago but nothing after that

**Comment by CodeTorso** [comment]
yep that endpoint uses llava-1.5-7b-hf, 

nowhere on cloudflare's docs we can get an idea of how much it will cost

but since we are already looking forward to adding pro plan, why not make it part of that ?

**Comment by Welding-Torch** [comment]
@CodeTorso As a user I would definitely expect this to be part of the free plan since importing tweets right now is basically our USP

**Comment by Dhravya** [comment]
Hmmmmm, llava right now is free. We can probably look into this and do a dual sort of import, should be not _that_ hard when we have the image URL that we can fetch

but we can only do this after the queues PR is merged, and then add an extra step in the add workflow to fetch any images and embed those too

**Comment by ameeetgaikwad** [comment]
@Dhravya we can add paid version for saving images. Can integrate stripe. What say?

**Comment by Welding-Torch** [comment]
Again,
As a user I would definitely expect recognizing content in saved tweet images to be part of the free plan since importing tweets right now is basically our USP

**Comment by ameeetgaikwad** [comment]
Hmm, that's right.

**Comment by MaheshtheDev** [comment]
this feature is already live with file import support
